frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -164,7 +181,11 @@ export const VMStatus: React.FC<VMStatusProps> = ({
   const vmiLike = vm || vmi;
 
   const { status, pod, progress, importerPodsStatuses } = vmStatusBundle;
-  const title = t(status.getLabelKey()) || status.toString() || t('kubevirt-plugin~Unknown'); // TODO status.toVerboseString() should be called to pass to popup header
+
+  const title =
+    t(status.getLabelKey()) ||
+    status.toString(getStatusSuffix(vmStatusBundle, t)) ||
```

Comment:
I believe you should call `t` on `status.toString` not inside it.  @yaacov wdyt

Reply:
I know it is not perfect but it is adhering to the recent convention. As I mentioned in https://github.com/openshift/console/pull/7725#discussion_r552608938 we can do a full refactoring but it is outside of the scope of this bug.